pkg/madmin/examples/add-service-account-and-policy.go

```diff
@@ -24,7 +24,10 @@ import (
 	"fmt"
 	"log"
 
+	"github.com/minio/minio/pkg/bucket/policy"
+	iampolicy "github.com/minio/minio/pkg/iam/policy"
 	"github.com/minio/minio/pkg/madmin"
+	"github.com/minio/minio/pkg/policy/condition"
```

Comment:
This should be 
```suggestion
	"github.com/minio/minio/pkg/bucket/policy"
	"github.com/minio/minio/pkg/bucket/policy/condition"
```
cause pkg/policy doesn't exist